tools/send-to-caas/bulkpublisher.html

```diff
@@ -88,6 +88,15 @@
                 <option>Dev</option>
               </select>
             </div>
+            <div class="field">
+              <label for="publishToFloodgate">Publish to Floodgate</label>
+              <select name="publishToFloodgate" id="publishToFloodgate">
+                <option value="default">--</option>
+                <option value="pink">Pink</option>
+                <option value="blue">Blue</option>
+                <option value="purple">Purple</option>
```

Comment:
I thought Pink was the only color used?

Reply:
@chrischrischris We recently received communication that we may need more than just pink. There are further changes needed in other parts as well to make the floodgate color a multiple choice, we will be raising separate PRs for those updates. Thanks!!

Reply:
@chrischrischris Yup, it will be just pink in the short term but it should be configurable. Akamai team said it is possible now to support multiple colors with the new edge-workers setup but the specific changes required to support it in Akamai will be discussed later when the requirement becomes current.

I am OK if the dropdown is limited to just pink for now as long as supporting additional colors is just about adding those additional options in the dropdown.

cc: @arshadparwaiz